templates/vite/src/App.tsx

```diff
@@ -0,0 +1,81 @@
+import { useConnectUI, useIsConnected } from "@fuels/react";
+import { useEffect } from "react";
+
+import { useRouter } from "./hooks/useRouter";
+import Button from "./components/Button";
+import Info from "./components/Info";
+import Wallet from "./components/Wallet";
+import Contract from "./components/Contract";
+import Predicate from "./components/Predicate";
+import Script from "./components/Script";
+import Faucet from "./components/Faucet";
+
+function App() {
+  const { connect } = useConnectUI();
+  const { isConnected, refetch } = useIsConnected();
+  const { view, views, setRoute } = useRouter();
+
+  useEffect(() => {
+    refetch();
+  }, [refetch]);
+
+  return (
+    <main
+      data-theme="dark"
+      className="flex items-center justify-center lg:pt-6 dark:text-zinc-50/90"
+    >
+      <div id="container" className="mx-8 mb-32 w-full max-w-6xl">
+        <nav id="nav" className="flex items-center justify-center py-1 md:py-6">
+          <a href="https://fuel.network/" target="_blank" rel="noreferrer">
+            <img src="./logo_white.png" alt="Fuel Logo" className="w-[124px]" />
+          </a>
+        </nav>
+
+        <div className="gradient-border rounded-2xl">
+          <div className="grain rounded-2xl p-1.5 drop-shadow-xl">
+            <div
+              id="grid"
+              className="lg:grid lg:grid-cols-7 lg:grid-rows-1 lg:gap-12"
+            >
+              <Info />
+              <div className="col-span-4">
+                <div className="gradient-border h-full rounded-xl bg-gradient-to-b from-zinc-900 to-zinc-950/80">
+                  {!isConnected && (
+                    <section className="flex h-full flex-col justify-center space-y-6 px-4 py-8 ">
+                      <Button onClick={() => connect()}>Connect Wallet</Button>
+                    </section>
+                  )}
+
+                  {isConnected && (
+                    <section className="flex h-full flex-col justify-center space-y-6 px-4 py-8">
```

Comment:
<img width="666" alt="image" src="https://github.com/user-attachments/assets/8577fb47-af2b-465b-b474-26224946d543">

I think there is too much going on with this tab view and the scrollbar is not ideal.

A suggestion would be to have only `contract`, `predicate` and `script` the tab frame. Then move the wallet and faucet related functionality to the left hand side?

Reply:
Agreed about the scrollbar. What do we think about a dropdown sort of a design/ an accordion almost?

Reply:
I personally like the tabs, but I think it should just be Contract, Predicate and Script.

I think the wallet and faucet should be on the left :)

Reply:
I fixed this problem by reducing the padding on the buttons, let me know what you guys think.

Reply:
This does solve the overflow issue.